### PR TITLE
Fix import to add createdAt and updatedAt for comments

### LINKED ISSUE #1555: 🐛 [Bug][Import] Comment dates are missing
> [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-12T13:52:56Z" title="Thursday, September 12th 2019, 3:52:56 pm +02:00">Sep 12, 2019</time>_
_Closed <time datetime="2019-09-13T18:37:34Z" title="Friday, September 13th 2019, 8:37:34 pm +02:00">Sep 13, 2019</time>_
---

## :bug: Bugreport

Comment dates are missing on all imported comments.

### Steps to reproduce the behavior
1. login into nitro-production
2. go on a imported post
3. see the comments list
4. there should be a date 

Bildschirmfoto 2019-09-12 um 12 00 03

### Expected behavior

Correct import of the comment dates.

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-13T11:51:20Z" title="Friday, September 13th 2019, 1:51:20 pm +02:00">Sep 13, 2019</time>_
_Merged <time datetime="2019-09-13T18:37:34Z" title="Friday, September 13th 2019, 8:37:34 pm +02:00">Sep 13, 2019</time>_
---

### Issues

- fix #1555 

Screenshot - 2019-09-13T140344 806

### Todo

- [X] None

### CODE REVIEW

File: deployment/legacy-migration/maintenance-worker/migration/neo4j/comments/comments.cql [hunk 1]
```diff
@@ -55,6 +55,8 @@ ON CREATE SET
 c.content        = comment.content,
 c.contentExcerpt = comment.contentExcerpt,
 c.deleted        = comment.deleted,
+c.createdAt      = comment.createdAt.`$date`,
+c.updatedAt      = comment.updatedAt.`$date`,
```
Comment (Tirokk) [inline deployment/legacy-migration/maintenance-worker/migration/neo4j/comments/comments.cql:59]: > [Tirokk](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-13T14:31:40Z" title="Friday, September 13th 2019, 4:31:40 pm +02:00">Sep 13, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -55,6 +55,8 @@ ON CREATE SET
 c.content        = comment.content,
 c.contentExcerpt = comment.contentExcerpt,
 c.deleted        = comment.deleted,
+c.createdAt      = comment.createdAt.`$date`,
+c.updatedAt      = comment.updatedAt.`$date`,
```


I suppose you have tested it.
Therefore I approve. 💫

But where is this `$date` coming from?

Comment (Tirokk) [reply deployment/legacy-migration/maintenance-worker/migration/neo4j/comments/comments.cql:59]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-13T18:30:11Z" title="Friday, September 13th 2019, 8:30:11 pm +02:00">Sep 13, 2019</time>_

---


the date would be coming from what is in the database in the `alpha`, I would guess... this is the same way we are setting the date for `Post`s

**Review by Mogge — APPROVED** [review]
I suppose you have tested it.
Therefore I approve. 💫

But where is this `$date` coming from?

**Review by Mogge — APPROVED** [review]
I've also tested the export/import and seen that it works in localhost
thanks @roschaefer, I was thinking of doing this tomorrow morning, but you've beaten me to it!!

**Pull request closed without merging**